recipes/highway/all/conanfile.py

```diff
@@ -0,0 +1,94 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class HighwayConan(ConanFile):
+    name = "highway"
+    description = "Performance-portable, length-agnostic SIMD with runtime " \
+                  "dispatch"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/google/highway"
+    topics = ("highway", "simd")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"fPIC": [True, False]}
```

Comment:
Do you know why they don't offer shared? Just curious, I didn't find any reference there

Reply:
Well, this library is almost header-only. There are only a few functions that are compiled. This is the reason, I think. Also, SIMD libraries are usually all header-only.